examples/images_contours_and_fields/interpolation_methods.py

```diff
@@ -27,14 +28,28 @@
 
 grid = np.random.rand(4, 4)
 
-fig, axs = plt.subplots(nrows=3, ncols=6, figsize=(9, 4.5),
+fig, axs = plt.subplots(nrows=3, ncols=6, figsize=(9.3, 6),
                         subplot_kw={'xticks': [], 'yticks': []})
 
-fig.subplots_adjust(hspace=0.3, wspace=0.05)
+fig.subplots_adjust(left=0.03, right=0.97, hspace=0.3, wspace=0.05)
```

Comment:
@timhoffm Concerning #11190, reducing the margins like I did here, may be a first option if the image is too large. I will not work in all cases though.